03-iam/README.md

```diff
@@ -366,11 +373,15 @@ _Is it possible to limit uploads of objects with a specific prefix (e.g.
 starting with "lebowski/") to an S3 bucket using IAM conditions? If not, how else
 could this be accomplished?_
 
+> not that i can tell
+
```

Comment:
You can do this - see https://aws.amazon.com/blogs/security/writing-iam-policies-grant-access-to-user-specific-folders-in-an-amazon-s3-bucket/ for how

Reply:
I did end up finding and implementing that, forgot to update/remove my comment !